src/views/Online/OnlineCreateGame.jsx

```diff
@@ -21,7 +22,8 @@ export function OnlineCreateGame({ ...props }) {
         </Menu>
       </div>
       <div className="p-4 h-full flex flex-col gap-4 items-center">
-        <h1>Creat game</h1>
+        <h1>Create game</h1>
+        <CreateGame />
         <Button
```

Comment:
Kann raus, da du cancel auch in CreateGame hast.

Reply:
XD, das ist mir nicht aufgefallen, bis ich deinen Kommentar hier gelesen habe. Ordentlicher Tunnelblick